internal/backends/winit/wasm_input_helper.rs

```diff
@@ -226,7 +226,7 @@ fn event_text(e: &web_sys::KeyboardEvent) -> Option<char> {
 
     let mut chars = key.chars();
     match chars.next() {
-        Some(first_char) if chars.next().is_none() => Some(first_char),
+        Some(first_char) if chars.next().is_none() => Some(first_char.into()),
```

Comment:
I think you can simplify this: no need to iterate over the chars, just  `Some(key.into())`  should work.